services/wallet/router.go

```diff
@@ -713,7 +713,7 @@ func (r *Router) suggestedRoutes(
 						continue
 					}
 
-					tx, err := bridge.BuildTx(network, addrTo, addrFrom, token, amountIn)
```

Comment:
These two parameters were reversed judging by the interface definition here https://github.com/status-im/status-go/blob/30e143ca40b68423edff1a54091c339ebc3694dc/services/wallet/bridge/bridge.go#L109
I'm not sure how everything else was working or all transfers were broken?